apps/server/src/routes/index.js

```diff
@@ -0,0 +1,10 @@
+const router = require('express').Router();
+const apiRoutes = require('./api');
+
+router.use('/api', apiRoutes);
```

Comment:
@AMess33 It looks like this file is your main entrypoint to your API routes but isn't being imported anywhere. The main entrypoint to your server is `main.js` which actually has this `/api` route defined there too. Might be worth just moving this logic over there. Notice in that file that your server runs at `http://localhost:3333`.

After that, I'd recommend using a tool called Postman to test out all your API routes to make sure you can hit them and they're doing what you expect. Here's a short intro to the tool, there's also a [VS Code extension](https://blog.postman.com/introducing-the-postman-vs-code-extension/) so you can make requests straight from your editor: https://www.youtube.com/watch?v=wmz1sGZp814

A couple things I'm noticing:
- doesn't look like your `createSession`, `updateSession`, and `deleteUser` functions are being used by any routes.
- you're creating multiple router instances with `require('express').Router()` across a few files, thinking that may cause issues. Once you've created it once, you can import/export the `router` where you need it to use the same instance everywhere.

Once you've verified you can hit all your routes and they're doing what you expect, you're ready to start hooking up the front-end! I'd recommend checking out [TanStack Query](https://tanstack.com/query/latest/docs/react/quick-start) for this part. It gives you some good patterns to follow for fetching data, handles caching and cache invalidation, and lets you build simple hooks to easily access pieces of data from anywhere in your app.

I think these issues are still relevant for this final piece, feel free to tweak the descriptions to be more accurate now that the app has evolved a bit and we've got more clarity around the app:
- https://github.com/AMess33/PokerRoll/issues/3
- https://github.com/AMess33/PokerRoll/issues/4
- https://github.com/AMess33/PokerRoll/issues/5